frontend/www/js/omegaup/arena/ranking.ts

```diff
@@ -25,127 +25,160 @@ interface Problem {
   visibility: number;
 }
 
+export interface RankingOptions {
+  startTime: Date | null;
+  finishTime: Date | null;
+  placesToShowInChart?: number;
```

Comment:
```suggestion
  placesToShowInChart: number;
```

abajo se le está poniendo un default, entonces no hay necesidad de declararlo opcional.

Reply:
Mismo comentario